src/main/java/nextstep/sessions/domain/data/registration/RegistrationInfo.java

```diff
@@ -0,0 +1,42 @@
+package nextstep.sessions.domain.data.registration;
+
+import nextstep.payments.domain.Payment;
+import nextstep.sessions.domain.data.session.Session;
+import nextstep.sessions.domain.data.session.UserPaymentInfo;
+import nextstep.users.domain.NsUser;
+
+public class RegistrationInfo {
+
+    private final Session session;
```

Comment:
db에 저장하기 위해 Session 에 대한 의존을 제거하지 못하셨을 것 같아요
id값만 참조하면 도메인간의 의존성을 제거할 수 있을 것 같아요
```suggestion
    private final Long sessionId;
```